website/docs/docs/on-premises/authentication/demo.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Demo
+---
+
+To allow experimentation with Pactflow without the need to configure an external identify provider, the "demo" authentication provider can be enabled. It will allow any user to log in to Pactflow with a name and email address.
```

Comment:
```suggestion
To allow experimentation with Pactflow without the need to configure an external identity provider, the "demo" authentication provider can be enabled. It will allow any user to log in to Pactflow with a name and email address.
```